### PR TITLE
deprecated include

### PR DESCRIPTION
**Pull request by bcoca**
goodby `include:`

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

include

### CODE REVIEW

**Review by ssbarnea — APPROVED** [review]

File: lib/ansible/modules/_include.py [hunk 1]
```diff
@@ -25,6 +25,11 @@
     directive (which was added in Ansible 2.1) to bypass autodetection.
   - This module is also supported for Windows targets.
 version_added: "0.6"
+deprecated:
+    why: it has too many conflicting behaviours depending on keyword combinations and it was unclear how it should behave in each case.
+        new actions were developed that were specific about each case and related behaviours.
+    version: "2.15"
+    alternatives: include_tasks, import_tasks, import_playbooks
```
Comment (flowerysong) [inline lib/ansible/modules/_include.py]:
```suggestion
    alternatives: include_tasks, import_tasks, import_playbook
```

**Comment by ssbarnea** [comment]
I can only say that I love seeing this fly away, a permanent source of confusions.

File: lib/ansible/modules/_include.py [hunk 2]
```diff
@@ -25,6 +25,12 @@
     directive (which was added in Ansible 2.1) to bypass autodetection.
   - This module is also supported for Windows targets.
 version_added: "0.6"
+deprecated:
+    why: it has too many conflicting behaviours depending on keyword combinations and it was unclear how it should behave in each case.
+        new actions were developed that were specific about each case and related behaviours.
+    alternative: include_tasks, import_tasks, import_playbook
+    removed_in: "2.15"
+    removed_from_colleciotn: 'ansbile.builtin'
```
Comment (mattclay) [inline lib/ansible/modules/_include.py]:
```suggestion
    removed_from_collection: 'ansbile.builtin'
```

File: test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 from voluptuous import ALLOW_EXTRA, PREVENT_EXTRA, All, Any, Invalid, Length, Required, Schema, Self, ValueInvalid
 from ansible.module_utils.six import string_types
 from ansible.module_utils.common.collections import is_iterable
+from ansible.release import __version__
```
Comment (mattclay) [inline test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py]: Since this is a bug fix for ansible-test, can you put it in a separate PR and provide a changelog?

Comment (bcoca) [reply test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py]: reverted the fix

**Comment by samdoran** [comment]
!waffling core_review
